### DISCUSSION TITLE
Bug: Dashboard UI display is broken

### DISCUSSION THREAD

**Discussion by brendanfalk** [aws/codewhisperer-command-line-discussions#34] [Support ticket]
### Checks

- [X] I ran `cw doctor` in the affected command line session and it didn't resolve my issue
- [X] I ran `cw restart` and it didn't resolve my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my issue

### Expected behaviour

Dashboard UI would display like a standard app

### Actual behaviour

![image](https://github.com/aws/codewhisperer-command-line-discussions/assets/4949076/adf43770-7afe-48e2-b7b6-43f56eec48fd)
![image](https://github.com/aws/codewhisperer-command-line-discussions/assets/4949076/871c2e31-5817-4ccb-b0d0-825c84b832c1)
<img width="568" alt="image" src="https://github.com/aws/codewhisperer-command-line-discussions/assets/4949076/2d349c97-7007-4346-b9cb-8580149104d7">

**Comment by brendanfalk** [comment]
From https://discord.com/channels/837809111248535583/1176573160569700372

**Comment by grant0417** [comment]
What version of macOS is this? The inspector/bar look quite old.

**Reply by greysonn** [reply]
haha, this is on a hackintosh build of macos mojave v10.14

**Comment by greysonn** [comment]
Greys-iMac:~ grey$ cw diagnostic

CODEWHISPERER-details:
  - 0.12.0
hardware-info:
  - model:
  - model-id:
  - chip-id: Intel(R) Core(TM) i7-9700K CPU @ 3.60GHz
  - cores: 8
  - mem: 32.00 GB
os-info:
  - macOS 10.14.4 (18E226)
environment:
  - shell: /bin/bash
  - terminal: iterm
  - cwd: /Users/grey
  - exe-path: /Users/grey/.local/bin/cw
  - install-method: brew
  - env-vars:
    - TERM: xterm-256color
    - SHELL: /bin/bash
    - CWTERM_SESSION_ID: 44abfee06c474dd2b9934215706103b4
    - CW_TERM: 0.12.0
    - PATH: /Users/grey/google-cloud-sdk/bin:/Library/Frameworks/Python.framework/Versions/3.11/bin:/Users/grey/.nvm/versions/node/v16.19.1/bin:/Users/grey/.yarn/bin:/Users/grey/.config/yarn/global/node_modules/.bin:/Users/grey/.pyenv/shims:/Users/grey/.pyenv/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/Applications/Wireshark.app/Contents/MacOS:/usr/local/go/bin:/Users/grey/go/bin:/Users/grey/.local/bin:/Users/grey/AppData/roaming/npm/node_modules/ngrok/bin:/Users/grey/.rvm/bin
    - CW_SET_PARENT_CHECK: 1
    - DISPLAY: /private/tmp/com.apple.launchd.YQGLyQ45bm/org.macosforge.xquartz:0

**Reply by grant0417** [reply]
Dont have a mac that old to test with, but will try to add CSS poly fills on the next release to ensure the CSS works as that seems to be the issue.

**Comment by brendanfalk** [comment]
Going to close out this ticket as we have only seen it on a hackintosh with an old mac which we understandably aren't supporting for the moment.
